faq.md

```diff
@@ -210,6 +210,29 @@ is to increase the NIC RX queue. See issue
 [#253](https://github.com/multipath-tcp/mptcp_net-next/issues/253) for more
 details.
 
+## Is GRO supported with MPTCP?
+
+Software Generic Receive Offload (GRO) will merge packets when the MAC headers
+are identical, and only a few TCP and IP headers can differ. In other words, if
+the TCP options of one packet following another one are identical, they can be
+merged.
+
+MPTCP adds TCP options in each packet, but when TCP Segmentation Offload (TSO)
+is used, the same MPTCP options will be duplicated in each packet, which is
+allowed by the specifications. When such packets are received, and if all the
```

Comment:
```suggestion
is used the same MPTCP options will be duplicated in each packet.
When such packets are received, and if all the
```

Reply:
Thanks! I kept the comma before "_the same MPTCP options will be duplicated in each packet_"